Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/FocusHandlerTrigger.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Utility component to hook up Unity events to the OnFocusEnter and OnFocusExit events.
+    /// </summary>
+    public class FocusHandlerTrigger : MonoBehaviour, IMixedRealityFocusHandler
```

Comment:
We tend to use the name "FocusHandler" instead of *Trigger, for example see "InputActionHandler"

Reply:
I like your name better, I'll change that.